pkg/cmd/issue/develop/develop.go

```diff
@@ -261,18 +260,11 @@ func checkoutBranch(opts *DevelopOptions, baseRepo ghrepo.Interface, checkoutBra
 			return err
 		}
 	} else {
-		gitFetch, err := opts.GitClient.Command(ctx.Background(), "fetch", "origin", fmt.Sprintf("+refs/heads/%[1]s:refs/remotes/origin/%[1]s", checkoutBranch))
-
+		err := opts.GitClient.Fetch(ctx.Background(), "origin", fmt.Sprintf("+refs/heads/%[1]s:refs/remotes/origin/%[1]s", checkoutBranch))
```

Comment:
Future idea for `GitClient.Fetch()` method: construct the refspec within the Fetch method itself so the caller does not have to (the caller just chooses which branches to fetch)